packages/plugin-koa/src/koa.js

```diff
@@ -90,15 +90,19 @@ module.exports = {
 }
 
 const getRequestAndMetadataFromCtx = ctx => {
-  const requestInfo = extractRequestInfo(ctx)
+  // Exclude new mappings from metaData but keep existing ones to preserve backwards compatibility
+  const { body, ...requestInfo } = extractRequestInfo(ctx)
```

Comment:
If we're trying to prevent the new mappings from being in metadata, should this also exclude `httpVersion`?

Reply:
Ah no, that was an existing thing in metadata so we can't remove it